theme_5/scss/_products.scss

```diff
@@ -0,0 +1,49 @@
+// Product
+
+.products-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 15px;
+}
+
+.products-list {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.product-link {
+	color: $black-solid;
+  top: 25px;
+
+	&:hover {
+		color: $black-solid;
+	}
+
+	&:active {
+		color: $black-solid;
+	}
+}
+
+.product-item {
+	border: 1px solid $cerise-red;
+	text-align: center;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered border, margin-bottom, text-align, width